src/main/scala/ch/epfl/bluebrain/nexus/kg/resolve/InAccountResolution.scala

```diff
@@ -0,0 +1,52 @@
+package ch.epfl.bluebrain.nexus.kg.resolve
+
+import cats.Monad
+import cats.instances.list._
+import cats.syntax.flatMap._
+import cats.syntax.traverse._
+import ch.epfl.bluebrain.nexus.kg.async.DistributedCache
+import ch.epfl.bluebrain.nexus.kg.resources._
+
+/**
+  * Resolution implementation that looks into every project belonging to a given account, in no particular order.
+  *
+  * TODO: ACLs
+  */
+class InAccountResolution[F[_]](accountRef: AccountRef, resources: Resources[F], cache: DistributedCache[F])(
```

Comment:
I think we should be consistent with the way we build those resolvers. `CrossProjectResolver` does not depend on `Projects` (or `DistributedCache `) as you called it. It just depends on a `List[Resolvers]` what gets passed to the constructor. In this way, this class does not have to know about the cache at all.

Reply:
It depends on how this class is to be used, and quite important. If we want to construct a single instance for an account and pass it around it's fine then, otherwise I agree with the above. But we should be consistent in the implementations.